packages/react-components/src/components/chart/converters/convertStyles.ts

```diff
@@ -37,6 +46,9 @@ export const getChartStyleSettingsFromMap =
  * @returns [StyleSettingsMap, (datastream: DataStream) => ChartStyleSettingsOptions]
  */
 export const useChartStyleSettings = (datastreams: DataStream[], chartOptions: ConvertChartOptions) => {
+  const highlightedDataStreams = useDataStore((state) => state.highlightedDataStreams);
+  const isDataStreamHighlighted = useDataStore((state) => state.isDataStreamHighlighted);
```

Comment:
usually, we do not use store to process data, like I am guessing this is not a action , this is a post-process on an action, so this may be better suited to be utility function rather than store action ?